irida_staramr_results/validate.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import logging
+import getpass
+import time
+
+from irida_staramr_results import util
+
+
+def user_credentials(username, password):
```

Comment:
Please add function docs for the 2 functions in this file that don't have them.

Reply:
Function docs added.